### ISSUE TITLE
Add parameter offset information to SS_output

### ISSUE THREAD

**Issue by chantelwetzel-noaa** [r4ss/r4ss#415]
It would be handy to add to the SS_output information of the setting of the parameter offset for the mortality and growth section.  Adding this information would facilitate automated code for summarized model output for inclusion in assessment documents (e.g., model parameter tables) that would present parameters in an understandable way (rather than males as offset from females) across models.  I can work to add this myself (unless @iantaylor-NOAA wants to do it first) but thought creating an issue for record would be useful for everyone.

**Comment by iantaylor-NOAA** [comment]
Good idea, @chantelwetzel-noaa. I would be happy to have you work on this.

I used the function below to replace the male M and Linf offset parameters with the resulting transformed values within the tables of parameter values for sensitivity analyses in the Big Skate assessment report.

However, I think something built into `SS_output()` should not do that replacement, but instead put the transformed values somewhere new. The most logical place among existing tables would be in `$MGparmAdj` from the table "MGparm_By_Year_after_adjustments" in Report.sso which has a column for every mortality and growth parameter showing values for each year. Adding the values there would be useful for any model with time-varying growth or mortality.

function from https://github.com/iantaylor-NOAA/BigSkate_Doc/blob/master/R/BigSkate_sensitivity_results.R#L48-L61
```
# function to convert offsets to standard space
convert.offsets <- function(tab){
  # male M offset
  tab[grep("NatM_p_1_Mal", tab$Label), -1] <-
    tab[grep("NatM_p_1_Fem", tab$Label), -1] *
      exp(tab[grep("NatM_p_1_Mal", tab$Label), -1])
  # male Linf offset
  # subset models for those that use male offset
  mods <- which(tab[grep("Linf_Mal", tab$Label), -1] < 0)
  tab[grep("Linf_Mal", tab$Label), 1 + mods] <-
    tab[grep("Linf_Fem", tab$Label), 1 + mods] *
      exp(tab[grep("Linf_Mal", tab$Label), 1 + mods])
  return(tab)
}  
```

**Comment by chantelwetzel-noaa** [comment]
I completely agree.  I think having an item in the SS_output list (i.e. $param_offset that would just read the setting as 0 or 1) could be used as a switch to create transformed parameters in the $MGparmAdj section.  Having a $param_offset switch could also be used by individuals in their own code determining where to grab mortality and growth parameters for males.

**Comment by iantaylor-NOAA** [comment]
Great. I see that there's an output like
```
MG_parms_Using_offset_approach_#:_1  (1=none, 2= M, G, CV_G as offset from female_GP1, 3=like SS2 V1.x)
```
at the end of the "Input_Variance_Adjustment" table in Report.sso between LIKELIHOOD and PARAMETERS, but that table isn't read by r4ss (as noted here: https://github.com/r4ss/r4ss/wiki/Matching-Report.sso-keywords-to-r4ss).

It's probably a good idea to read the whole "Input_Variance_Adjustment" table in `SS_output()` and then separate that final line into a new variable as you suggest (but maybe it should be called `$MG_parms_offset` to better match Report.sso.

**Comment by iantaylor-NOAA** [comment]
This idea from @chantelwetzel-noaa would be good to have available for the 2021 U.S. west coast assessments, so I've created a new "2021 U.S. west coast assessments" milestone and added this to it.
